src/meta/client/MetaClient.cpp

```diff
@@ -190,6 +191,11 @@ bool MetaClient::loadData() {
             return false;
         }
 
+        if (!loadPlugins(pluginItems)) {
```

Comment:
1.  You don't need to load plugins for each space.
2.  After loading the new plugins,  you should open them. Otherwise we have to restart the service.

Reply:
good catch, I will fix it

Reply:
has fixed.